nrf_wifi/fw_if/umac_if/src/fmac_api_common.c

```diff
@@ -1079,6 +1079,19 @@ int nrf_wifi_phy_rf_params_init(struct nrf_wifi_osal_priv *opriv,
 	prf->phy_params[rf_param_offset + 28]  = CONFIG_NRF700X_ANT_GAIN_5G_BAND2;
 	prf->phy_params[rf_param_offset + 29]  = CONFIG_NRF700X_ANT_GAIN_5G_BAND3;
 
+#ifdef CONFIG_NRF700X_RADIO_TEST
+	prf->temp_volt_backoff.max_chip_temp = 0;
+	prf->temp_volt_backoff.min_chip_temp = 0;
+	prf->temp_volt_backoff.lb_max_pwr_bkf_hi_temp = 0;
+	prf->temp_volt_backoff.lb_max_pwr_bkf_low_temp = 0;
+	prf->temp_volt_backoff.hb_max_pwr_bkf_hi_temp = 0;
+	prf->temp_volt_backoff.hb_max_pwr_bkf_low_temp = 0;
+	prf->temp_volt_backoff.lb_vbt_lt_vlow = 0;
+	prf->temp_volt_backoff.hb_vbt_lt_vlow = 0;
+	prf->temp_volt_backoff.lb_vbt_lt_low = 0;
+	prf->temp_volt_backoff.hb_vbt_lt_low = 0;
```

Comment:
Why is it needed to set some values to 0 specifically. Why can't `prf` be initialized to all `0` at the beginning of the function and only the specific values updated later in the function?